src/stores/events/EventsStore.ts

```diff
@@ -80,6 +80,9 @@ export default class EventsStore {
 
 	@observable.ref selectedNode: EventTreeNode | null = null;
 
+	@observable
```

Comment:
let make it observable.ref, event properties won't change and we dont need to observe them